python/numpy/exercicesolution.py

```diff
@@ -71,7 +71,7 @@ def ex4_2_alt():
 
 def ex5_inefficient_fill(height=1000, width=1000):
     """Inefficient fill using 2 for loops"""
-    data = numpy.zeros((height, width), dtype=numpy.float)
+    data = numpy.zeros((height, width), dtype=float)
```

Comment:
maybe it would be wiser to replace it by `np.float32` ? I would say a good practice would be to use those types directly

Reply:
+1 float is a python type. Specifying `np.float64` or leaving it empty (which defaults to `float64`)

Reply:
I don't see the issue in using Python types: `np.dtype(float) is np.dtype(np.float64)` and `np.dtype(int) is np.dtype(np.int64)`.

Reply:
the idea was to get closer to 'best practices' and maybe avoid later some error like type being different from one platform to the other. But fine with me if you prefer it this way.